tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -153,6 +153,42 @@ private int Foo(int one, int two, int three)
         }
 
         [Fact]
+        public async Task AttributeConstructorSignatureHelp()
+        {
+            // 1st position, a
+            const string source =
+                @"using System;
+
+        namespace New_folder
+        {
+            using System;
+        [MyTest($$)]
+        public class TestClass {
+            int value;
+            public static void Main(){
+            }
+            void TestMethod(int value){
+                this.value = value;
+            }
+            public class MyTestAttribute : Attribute {
```

Comment:
newline here

Reply:
C# code typically puts opening `{` on a new line. Consider using VS to format the document as this will get your style in-line with C#.